dist/.magento.env.yaml

```diff
@@ -262,6 +262,28 @@
 #                elasticsearch_index_prefix: magento                                                                  #
 #                elasticsearch_server_timeout: '15'                                                                   #
 #######################################################################################################################
+# ELASTICSUITE_CONFIGURATION - use this variable to retain customized ElasticSuite service settings                   #
+# between deployments. All configs provided in this configuration are nested in main ElasticSuite                     #
+# config section 'system/default/smile_elasticsuite_core_base_settings'                                               #
+# Magento Version: 2.2.0 and later                                                                                    #
+# Default value: Depends on template configuration of ElasticSearch (i.e. default number of replicas and shards)      #
```

Comment:
Would be good to mention _merge here. Or mention it in additional section for all such configs

Reply:
That's general question to Docs Team